nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/ChartProcessor.java

```diff
@@ -963,10 +964,29 @@ boolean scriptHasDataset(String script) {
             String filePath = getNMRData().getFilePath();
             File nmrFile = new File(filePath);
             File directory = nmrFile.isDirectory() ? nmrFile : nmrFile.getParentFile();
-            FileChooser fileChooser = new FileChooser();
-            fileChooser.setInitialDirectory(directory);
-            fileChooser.setInitialFileName(datasetName);
-            File file = fileChooser.showSaveDialog(null);
+            File file;
+            if (getDatasetType().equals("SPINit")) {
+                Path procDir = Path.of(directory.toString(),"Proc");
+                int procNum;
+                try {
+                    procNum = RS2DData.findLastProcNum(procDir).orElse(0);
+                } catch (IOException e) {
+                    procNum = 0;
+                }
+                String procNumStr = GUIUtils.input("Proc Number",String.valueOf(procNum)).trim();
+                File procNumDir = Path.of(directory.toString(),"Proc",procNumStr).toFile();
```

Comment:
Now autoincrements (with your changes).  I wonder if we want to add a GUI setting to allow overwriting vs. incrementing.  With small datasets it may makes sense to always increment, but with large 3D or 4D experiments you could end up with multiple copies of files that are each hundreds of MBs